solution.cpp

```diff
@@ -1,23 +1,152 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright &copy; 2015 Duy Nguyen <andy21996@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
-/**
- * TODO: program description
- */
+// This program will convert integer number into Roman numeral from 1 to 4000
+#include "stdafx.h"
```

Comment:
i've probably mentioned this to you in lab by now, but `stdafx.h` is a windows only thing -- best to avoid it, unless for some reason you really do need it :-)